### ISSUE TITLE
Can this scheme be made distributed?

### ISSUE THREAD

**Issue by Fiono11** [scipr-lab/poly-commit#6]
Instead of a trusted dealer generating the UniversalParams, can they be generated in a distributed way?

**Comment by burdges** [comment]
Yes, doing so is a straightforward sequential trusted setup.  Each stage proves correctness using `E::pairing(params.powers_of*_g[i],params.h) = E::pairing(params.powers_of*_g[i+1],E::G2Affine::prime_subgroup_generator())` etc, although Schnorr DLEQ proofs sound faster.

**Comment by Fiono11** [comment]
I'm sorry, I didn't understand. Can you explain in a simpler way, please?

**Comment by Pratyush** [comment]
Yes, one can essentially just modify the Powers of Tau setup to make this work.

**Comment by Fiono11** [comment]
Can you explain it, please?
